.github/workflows/testAndPublish.yml

```
@@ -175,6 +172,7 @@ jobs:
     - name: Install the latest version of uv
       uses: astral-sh/setup-uv@v6
```

Won't this step still run even if `feature_crowdinSync` is not set?
```suggestion
    - name: Install the latest version of uv
      if: ${{ env.feature_crowdinSync }}
      uses: astral-sh/setup-uv@v6
```